assets/js/modules/analytics-4/datastore/settings.test.js

```diff
@@ -521,27 +571,266 @@ describe( 'modules/analytics-4 settings', () => {
 						isScriptAccessEnabled: true,
 					} );
 
-				const fpmSettingsEndpoint = new RegExp(
-					'^/google-site-kit/v1/core/site/data/fpm-settings'
-				);
+				const validSettings = {
+					accountID: fixtures.createProperty._accountID,
+					propertyID: fixtures.createProperty._id,
+					webDataStreamID: fixtures.createWebDataStream._id,
+				};
+
+				registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.setSettings( validSettings );
+
+				fetchMock.postOnce( settingsEndpoint, {
+					body: validSettings,
+					status: 200,
+				} );
 
 				fetchMock.postOnce( fpmSettingsEndpoint, {
-					body: JSON.stringify( {
-						data: { settings: { isEnabled: true } },
-					} ),
+					body: error,
+					status: 500,
+				} );
+
+				registry.dispatch( CORE_SITE ).setFirstPartyModeEnabled( true );
+				const { error: submitChangesError } = await registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.submitChanges();
+
+				expect( submitChangesError ).toEqual( error );
+
+				expect( console ).toHaveErrored();
+			} );
+
+			it( 'should not send a POST request to the FPM settings endpoint when the toggle state is changed', async () => {
```

Comment:
As above, I added a couple of additional tests to extend the coverage for the existing behaviour.